cvat/apps/iam/views.py

```diff
@@ -143,9 +180,9 @@ def post(self, request, *args, **kwargs):
             self.serializer.is_valid(raise_exception=True)
         except ValidationError:
             user = self.serializer.get_auth_user(
-                self.serializer.data.get('username'),
-                self.serializer.data.get('email'),
-                self.serializer.data.get('password')
+                self.serializer.data.get("username"),
```

Comment:
A lot of changes in the file. Could you please revert them? Why do we need them?

Reply:
I applied black for this file, I'll revert